### DISCUSSION TITLE
Unable to change the `primary` color

### DISCUSSION THREAD

**Discussion by nilsenpaul** [filamentphp/filament#12993] [Help]
### Package

Other

### Package Version

v3.0.0

### How can we help you?

In `AppServiceProvider.php`, I added:

```
FilamentColor::register([
  'primary' => Color::Teal,
]);
```

In the admin panel, elements with class `text-primary` still show in the 'default' amber color. When I change `gray` for instance, it does work. How do I get the change to reflect to my admin panel?

**Comment by leandrocfe** [comment]
add in your panel provider
```php
return $panel
...
->colors([
    'primary' => Color::Teal
])
```
